sqlagg/base.py

```diff
@@ -73,6 +73,10 @@ def execute(self, metadata, connection, filter_values):
         query = self._build_query(metadata)
         return connection.execute(query, **filter_values).fetchall()
 
+    def count(self, metadata, connection, filter_values):
+        query = self._build_query(metadata).count()
```

Comment:
I assume you've tested this but I don't see this kind of usage of `count` in the SQLAlchemy docs. Does this still work if the query has a `limit` or `group by` applied?